purchases/src/main/java/com/revenuecat/purchases/interfaces/GetStoreProductsCallback.java

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.interfaces;
+
+
+import com.revenuecat.purchases.PurchasesError;
+import com.revenuecat.purchases.models.StoreProduct;
+
+import java.util.List;
+
+public interface GetStoreProductsCallback {
```

Comment:
Hmm interesting... Have you tried using the `@JvmSuppressWildcards` annotation? https://kotlinlang.org/docs/java-to-kotlin-interop.html#null-safety

Since we should not be exposing the implementations right now, I think that should be ok.